lib/postgrex/connection.ex

```diff
@@ -140,33 +172,45 @@ defmodule Postgrex.Connection do
 
   ## Options
 
-    * `:timeout` - Call timeout (default: `#{@timeout}`)
+    * `:pool_timeout` - Time to wait in the queue for the connection
+    (default: `#{@pool_timeout}`)
+    * `:queue` - Whether to wait for connection in a queue (default: `true`);
+    * `:timeout` - Execute request timeout (default: `#{@timeout}`);
     * `:decode`  - Decode method: `:auto` decodes the result and `:manual` does
     not (default: `:auto`)
+    * `:decode_mapper` - Fun to map each row in the result to a term, see
```

Comment:
Do we care about making this public? Would using a mapper offer any advantage over `decode: :manual`? If so, maybe we should get rid of `decode: :manual` altogether?